File: Euler 2/Euler 2.py
__author__ = 'eddie'

def fibo(n):
    sum = 0
    a, b= 0 , 1
    c = a + b
    while c < n:
        if c % 2 == 0:
            sum = sum + c
        a, b = b , c
        c = a + b

    return sum

print fibo(4000000)

File: 002/Euler2.js
(function euler2() {
	var sum = 0;
	var a = 0;
	var b = 1;
	var c;
	for (c = a + b; c < 4000000;){
		c = a + b;
		a = b;
		b = c;
		if (c % 2 === 0){
			sum += c;
		}
	}
	console.log(sum);
	return sum;
})();

